src/settings-ui/Settings.UI/Flyout/LaunchPage.xaml

```diff
@@ -10,157 +10,137 @@
+                <AppBarButton
                     x:Name="DocsBtn"
                     x:Uid="DocsBtn"
-                    Click="DocsBtn_Click"
-                    Style="{StaticResource FlyoutButtonStyle}">
+                    Icon="Document"
+                    Click="DocsBtn_Click">
                     <ToolTipService.ToolTip>
-                        <TextBlock x:Uid="DocsTooltip"/>
+                        <ToolTip x:Uid="DocsTooltip"/>
                     </ToolTipService.ToolTip>
-                    <FontIcon FontSize="14" Glyph="&#xE130;" />
-                </Button>
-                <Button
+                </AppBarButton>
+                <AppBarButton
                     x:Name="ReportBugBtn"
                     x:Uid="BugReportBtn"
-                    Click="ReportBugBtn_Click"
-                    Style="{StaticResource FlyoutButtonStyle}">
+                    Click="ReportBugBtn_Click">
+                    <AppBarButton.Icon>
+                        <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xEBE8;"/>
+                    </AppBarButton.Icon>
                     <ToolTipService.ToolTip>
-                        <TextBlock x:Uid="BugReportTooltip"/>
+                        <ToolTip x:Uid="BugReportTooltip"/>
                     </ToolTipService.ToolTip>
-                    <FontIcon FontSize="14" Glyph="&#xebe8;" />
-                </Button>
-                <Button
+                </AppBarButton>
+                <AppBarButton
                     x:Name="SettingsBtn"
-                    Padding="8"
                     x:Uid="SettingsBtn"
                     Click="SettingsBtn_Click"
-                    Style="{StaticResource FlyoutButtonStyle}">
+                    Icon="Setting">
                     <ToolTipService.ToolTip>
-                        <TextBlock x:Uid="SettingsTooltip"/>
+                        <ToolTip x:Uid="SettingsTooltip"/>
                     </ToolTipService.ToolTip>
-                    <AnimatedIcon x:Name="SearchAnimatedIcon">
-                        <AnimatedIcon.Source>
-                            <animatedVisuals:AnimatedSettingsVisualSource />
-                        </AnimatedIcon.Source>
-                        <AnimatedIcon.FallbackIconSource>
-                            <SymbolIconSource Symbol="Setting" />
-                        </AnimatedIcon.FallbackIconSource>
-                    </AnimatedIcon>
-                </Button>
-                <!--<AppBarSeparator />
-                <Button
-                    x:Name="QuitBtn"
-                    Style="{StaticResource FlyoutButtonStyle}"
-                    ToolTipService.ToolTip="Quit"
-                    Click="QuitButton_Click">
-                    <FontIcon FontSize="14" Glyph="&#xe894;" />
-                </Button>-->
-            </StackPanel>
-        </Grid>
+                </AppBarButton>
+                <AppBarSeparator />
+                <AppBarButton
```

Comment:
I think we wanted to leave the quit button this out of the flyout.. but I can't remember why. Wasn't there some sort of bug that we ran into @jaimecbernardo ?